blade-graphics/src/gles/command.rs

```diff
@@ -458,6 +468,38 @@ impl crate::VertexFormat {
     }
 }
 
+impl crate::BlendFactor {
+    fn describe(self) -> u32 {
```

Comment:
can we call this method `to_gles`?

Reply:
This is named like `VertexFormat::describe` defined above, but `to_gles` seems to be the better name. Should the function above also be renamed to `VertexFormat::to_gles`?

Reply:
`VertexFormat` is a bit different since it actually describes a format by producing a bunch of characteristics (which are GLES specific), it's not a 1:1 translation like all the rest. I don't have a strong opinion about how `VertexFormat::describe` should be named, leaving up to you.